cmd/containerd/command/main_unix.go

```diff
@@ -45,12 +45,14 @@ func handleSignals(ctx context.Context, signals chan os.Signal, serverC chan *se
 			case s := <-serverC:
 				server = s
 			case s := <-signals:
+				if s == unix.SIGPIPE {
+					continue
+				}
+
```

Comment:
Without a comment describing why this special case is here, there's a risk that someone will remove this.

What if, instead, we'd move the `log.G(...` inside the `switch` ?

Reply:
Hi @thaJeztah sorry for this late.  I add some comments on my change.  Move `log.G(..` inside the `swith` may cause some depluted code?  how do you think about that?